src/com/interface.rs

```diff
@@ -1,58 +1,56 @@
 use crate::*;
 
-/// A COM interface
+/// A COM interface.
 ///
 /// # Safety
 /// Implementors of this trait must be transparent wrappers
-/// around pointers to the associated VTable for the interface.
+/// around pointers to the associated vtable for the interface.
 /// In other words, implementators must be exactly equivalent to
-/// *mut *mut Self::VTable.
-///
-/// VTables must be COM compliant VTables where the first three function
-/// pointers are the IUknown methods.
-///
-/// Additionally, because ComInterfaces are just pointers to vtables,
-/// it must be safe to zero initialize the interface.
+/// `*mut *mut Self::VTable`.
 pub unsafe trait ComInterface: Sized {
+    /// `VTable` must be a COM compliant vtable where the first three function
+    /// pointers are the `IUnknown` methods. And because ComInterfaces are just
```

Comment:
This last sentence is about ComInterfaces as a whole not about vtables. I think we should move it back up

Reply:
Will do.